content/_data/changelogs/weekly.yml

```diff
@@ -12426,6 +12427,62 @@
   # pull: 5681 (PR title: Bump spotless-maven-plugin from 2.12.2 to 2.12.3)
   # pull: 5695 (PR title: Update at-since up to 2.308)
 
+  - version: '2.310'
+    date: 2021-09-06
+    changes:
+      - type: TODO
+        category: developer
+        pull: 5703
+        authors:
+          - dependabot[bot]
+        pr_title: Bump jnr-posix from 3.1.7 to 3.1.8
+        message: |-
+          (No proposed changelog)
```

Comment:
This changelog entry should skipped because other patch revisions of jnr-posix were also skipped
```suggestion
```

Reply:
I see you can added the label, you can re-trigger the changelog generation from https://github.com/jenkinsci/jenkins/actions/workflows/changelog.yml